src/app/components/elements/PageTitle.tsx

```diff
@@ -150,7 +150,7 @@ export const PageTitle = ({currentPageTitle, displayTitleOverride, subTitle, dis
             <meta property="og:title" content={currentPageTitle} />
         </Helmet>
         {audienceViews && <AudienceViewer audienceViews={audienceViews} />}
-        {help && !showModal && <React.Fragment>
+        {isAda && help && !showModal && <React.Fragment>
```

Comment:
Removing `modalId` means we should also remove `showModal`
```suggestion
        {isAda && help && <React.Fragment>
```

We should also remove the original showModal definition (L104), this page's HelpModal (L119-134) and the Fragment below this (L157-159). None of that code has any use here anymore.